ci-operator/jobs/infra-periodics.yaml

```diff
@@ -801,6 +801,46 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+    workdir: true
+  interval: 6h
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-branch-protector-openshift-org
+  spec:
+    containers:
+    - args:
+      - --config-path=./core-services/prow/02_config/_config.yaml
+      - --supplemental-prow-config-dir=./core-services/prow/02_config
+      - --job-config-path=./ci-operator/jobs
+      - --confirm=true
+      - --github-token-path=/etc/github/oauth
+      - --github-endpoint=http://ghproxy
+      - --github-endpoint=https://api.github.com
+      - --github-hourly-tokens=3000
```

Comment:
Does this implicitly increase the maximum token usage for the `openshift` org from 1k to 3k per hour?

Reply:
Yes, but on the merge-robot account instead of the app. The 3k is what we used to have for the whole branchprotector job before we switched it to apps auth: https://github.com/openshift/release/commit/6d03fe2a051460870e47b4376edb3c440c0fdcfb

/hold cancel